travis_build.sh

```diff
@@ -126,22 +131,19 @@ fi
 
 if [[ $MODE = 'genGenTests' ]] ; then
   echo "Doing 'sbt genGenTests/test'"
-  export SBT_OPTS="-server -Xms1G -Xmx2G -Xss10M -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -XX:NewRatio=8 -XX:MaxPermSize=512M -XX:-UseGCOverheadLimit"
```

Comment:
> And in each of the `MODE` builds of `travis_build.sh`

I'm not finding anything compelling about all these, so I've deleted them.  This makes it easier to get the JDK6 hack to work here.